### DISCUSSION TITLE
Can we control VITS's seed during inference ?

### DISCUSSION THREAD

**Discussion by theodorblackbird** [coqui-ai/TTS#1475] [General Q&A] [answered]
Hi,
VITS makes use of density estimation, for instance with the duration predictor. When I use VITS on the same sentence multiple times, I can't feel that a big difference between two inferences. Is that a normal behaviour or due to some fixed seed ? In all cases : can we set manually the seed ?
Thanks

**Comment by erogol** [comment, marked as answer]
Try larger ```inferece_noise_scale``` values in the config
